components/locid/src/extensions/other/mod.rs

```diff
@@ -86,7 +85,7 @@ impl Other {
 
         let mut keys = ShortBoxSlice::new();
         while let Some(subtag) = iter.peek() {
-            if !Subtag::valid_key(subtag) {
+            if subtag.len() < 2 {
```

Comment:
Question: why did this change?

Reply:
good call, I missed it somehow during my refactors. Fixing!